content/en/community/contributing/code/static-analysis.md

```diff
@@ -8,11 +8,16 @@ aliases: >
   /contribute/code/static-analysis
 ---
 
-## eslint
+## ESLint
 
-All code must pass an eslint check which runs early in the CI cycle and uses the [standard medic eslint configuration](https://github.com/medic/eslint-config).
+All code must pass an eslint check which runs early in the CI cycle and uses the [standard medic ESLint configuration](https://github.com/medic/eslint-config).
 
-ESLint has [plugins/integrations](https://eslint.org/docs/latest/use/integrations) for many IDEs that will report linting issues in real-time.
+### Linting
+
+There's two main options for linting locally before you push your code, at which point CI will lint it.  It's nice to lint before you push because you get near instant feedback where as CI might take 30 minutes or more:
```

Comment:
Technically, we should be linting right at the beginning of the CI run, so it should fail much faster than "30 minutes".  However, for first time contributors, the CI will not run until it is approved.  In that case, it could be several days before the CI is run.  

My preference would be to update this text to just say that you should always lint the code before committing.

Reply:
ah - fair!

fixed in 2798ade2946aa9c9cffabee76689971c13226285